### PR TITLE
Remove consensus

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/neo/Plugins/IConsensusProvider.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+using Neo.Wallets;
+
+namespace Neo.Plugins
+{
+    public interface IConsensusProvider
+    {
+        void Start(Wallet wallet);
```
Comment (shargon) [inline src/neo/Plugins/IConsensusProvider.cs:7]: Add `Stop` and `IsWatchOnly`?

Comment (erikzhang) [reply src/neo/Plugins/IConsensusProvider.cs:7]: Why a consensus node need to stop?

Comment (erikzhang) [reply src/neo/Plugins/IConsensusProvider.cs:7]: `IsWatchOnly` can be easily checked if you have the wallet.

**Review by shargon — APPROVED** [review]

File: src/neo/Plugins/IConsensusProvider.cs (hunk 1 above)
Comment (vncoelho) [reply src/neo/Plugins/IConsensusProvider.cs:7]: Sounds good, Erik

**Review by vncoelho — APPROVED** [review]

**Review by bibicoco2015 — COMMENTED** [review]
like

**Pull request merged**